Blog - Istio at KubeCon EU (#13122)
istio/istio.io: content/en/blog/2023/istio-at-kubecon-eu/index.md

---

> title: "Istio at KubeCon Europe 2023!"

Reviewer: ```suggestion
title: "Istio at KubeCon Europe 2023"
```

> description: A quick recap of Istio at KubeCon Europe, RAI, Amsterdam, The Netherlands.

Reviewer: ```suggestion
description: A quick recap of Istio at KubeCon Europe, at the RAI in Amsterdam.
```

publishdate: 2023-04-27
attribution: "Faseela K, for the Istio Day Programme Committee"
keywords: [Istio Day,IstioCon,Istio,conference,KubeCon,CloudNativeCon]
---

> The open source and cloud native community gathered from 18th to 21st April in Amsterdam for the first KubeCon of 2023. The four-day conference, organized by the Cloud Native Computing Foundation, was special for Istio, as we evolved from a **ServiceMeshCon** to our **first official project co-located event**.

Reviewer: ```suggestion
The open source and cloud native community gathered from 18th to 21st April in Amsterdam for the first KubeCon of 2023. The four-day conference, organized by the Cloud Native Computing Foundation, was special for Istio, as we evolved from a participant at ServiceMeshCon to hosting our first official project co-located event.
```

Istio’s presence at the conference did not end with Istio Day. The second day keynote started with a [project update video](https://twitter.com/linsun_unc/status/1648952723604221953) from Lin Sun. It was also a proud moment for us, when our steering committee member Craig Box was [recognized as a CNCF mentor](https://twitter.com/IstioMesh/status/1648722572366708739) in the keynote. The maintainer track for Istio presented by TOC member Neeraj Poddar grabbed great attention as he talked about the current ongoing efforts and future roadmap of Istio. The talk, and the size of the audience, underlined why Istio continues to be the most popular service mesh in the industry.

{{< image width="75%"
    link="./use-istio-in-production.jpg"
    caption="KubeCon Europe 2023, Question: How many of you use Istio in production?"
    >}}

> The following sessions at KubeCon were based on Istio and almost all of them had huge crowd in attendance:

Reviewer: ```suggestion
The following sessions at KubeCon were based on Istio and almost all of them had a huge crowd in attendance:
```

* [Future of Istio - Sidecar, Sidecarless or Both?](https://sched.co/1HySB)
* [Operate Multi Tenancy Istio with ArgoCD in production](https://sched.co/1Hyd1)
* [Create Istio Filters with Any Programming Language](https://sched.co/1HybK)
* [Automated Cloud-Native Incident Response with Kubernetes and Service Mesh](https://sched.co/1HyZ9)
* [Autoscaling Elastic Kubernetes Infrastructure for Stateful Applications Using Proxyless gRPC and Istio](https://sched.co/1HyXz)
* [Developing a Mental Model of Istio: From Kubernetes to Sidecars to Ambient](https://sched.co/1HyZj)
* [Future of ServiceMesh - Sidecar, Sidecarless or Proxyless? - Panel Discussion](https://sched.co/1Hydb)
* [The Top 10 List of Istio Security Risks and Mitigation Strategies](https://sched.co/1HyPQ)

> Istio had a full time kiosk in the KubeCon project pavilion, with the majority of questions asked being on the status of our CNCF graduation.
> 
> {{< image width="75%"
>     link="./istio-booth.jpg"
>     caption="KubeCon Europe 2023, Istio Kiosk"
>     >}}
> 
> We are so excited to know that our users are eagerly waiting for news of our graduation, and we promise we are actively working towards it! Many of our TOC members and maintainers also offered support at the booth, where a lot of interesting discussions happened around Istio Ambient Mesh as well.

Reviewer: ```suggestion
Istio had a full time kiosk in the KubeCon project pavilion, with the majority of questions asked being on the status of our CNCF graduation. We are so excited to know that our users are eagerly waiting for news of our graduation, and we promise we are actively working towards it!

{{< image width="75%"
    link="./istio-booth.jpg"
    caption="KubeCon Europe 2023, Istio Kiosk"
    >}}

Many of our TOC members and maintainers also offered support at the booth, where a lot of interesting discussions happened around Istio Ambient Mesh as well.
```

{{< image width="75%"
    link="./toc-members-at-kiosk.jpg"
    caption="KubeCon Europe, More support at Istio Kiosk"
    >}}

Another highlight was Istio TOC and steering members and authors Lin Sun and Christian Posta signing copies of the "Istio Ambient Explained" book.

{{< image width="75%"
    link="./ambient-mesh-book-authors.jpg"
    caption="KubeCon Europe 2023, Ambient Mesh book signing by authors"
    >}}

Last, but not least, we would like to express our heartfelt gratitude to our platinum sponsors [Tetrate](http://tetrate.io/), for supporting Istio Day!

> 2023 is going to be really big for Istio, with more events planned for the coming months! Stay tuned for updates on IstioCon 2023 and Istio Day at KubeCon NA.

Reviewer: ```suggestion
2023 is going to be really big for Istio, with more events planned for the coming months! Stay tuned for updates on IstioCon 2023 and Istio's presence at KubeCon in China and North America.
```
